geomstats/lie_group.py

```diff
@@ -146,10 +146,26 @@ def group_log(self, point, base_point=None, point_type=None):
             point_type = self.default_point_type
 
         identity = self.get_identity(point_type=point_type)
-        identity = self.regularize(identity, point_type=point_type)
+        #identity = self.regularize(identity, point_type=point_type)
```

Comment:
left-over comment?

Reply:
I thought it was weird to regularise an identity point... commented it out for the time being, but yes it should be removed.